src/pages/Index/index.tsx

```diff
@@ -72,6 +72,7 @@ export class IndexBase extends React.Component<Props> {
 
   componentDidMount() {
     const { dispatch } = this.props;
+    dispatch(versionsActions.unsetCurrentBaseVersionId());
```

Comment:
I guess it might not be worth it if this is the only place we're going to need to call both, but I was wondering if it would be worth having an action like `unsetCurrentVersions` which does both `unsetCurrentBaseVersionId` and `unsetCurrentVersionId`?

Reply:
I'd say it's probably not worth it